package.json

```diff
@@ -23,8 +23,7 @@
   ],
   "author": "Rain Agency <npm@rain.agency> (http://rain.agency)",
   "engines": {
-    "node": "4.3.2",
-    "node": "6.10"
+    "node": ">=4.3.2 <=6.10"
```

Comment:
Do we really need to restrict the maximum engine? I don't think we're that likely to break on higher versions.